v2/src/core/components/AppTransformHash.vue

```diff
@@ -27,16 +29,30 @@ const props = defineProps({
     italic: {
         type: Boolean,
         default: false
+    },
+    start: {
```

Comment:
we can probably replace this with one variable ( like is isShort: boolean) then we dont have to use them both